docs/src/components/mdx/index.module.css

```diff
@@ -16,7 +16,8 @@ main > .h3 {
   padding: var(--psLayoutSpacingXXSmall) 0 var(--psLayoutSpacingXXSmall)
     var(--psLayoutSpacingMedium);
 }
-.headingLink {
+
+:not(#fakeid-to-make-more-specific-than-glamor-styles) .headingLink {
```

Comment:
oh my, I've ever seen this hake before.  lol, that's ridiculous and awesome.

feels like we're back to putting stars before selectors days. :D

Reply:
yolo